packages/graphql/src/schema/get-custom-resolver-meta.test.ts

```diff
@@ -43,14 +66,17 @@ describe("getCustomResolverMeta", () => {
                     name: { value: "RANDOM 4" },
                 },
             ],
+            name: {
+                kind: Kind.NAME,
+                value: fieldName,
+            },
         };
 
-        const result = getCustomResolverMeta(field);
+        const result = getCustomResolverMeta(field, object, resolvers);
 
         expect(result).toBeUndefined();
     });
-
-    test("should throw if requires not a list", () => {
+    test("should throw if requires not a list 123", () => {
```

Comment:
I don't understand this rename :thinking:

Reply:
Ah yes, this test name was the same as the start of another so wasn't skipping. I obviously meant to come up with a better name though! Should be fixed now 😄